### PR TITLE
Feature setplaymode

### PR DESCRIPTION
**Pull request by ifcodesdotnet**
I've added some extra code to expose setting a play mode!

### CODE REVIEW

File: src/ByteDev.Sonos.Upnp/Services/AvTransportService.cs [hunk 1]
```diff
@@ -168,5 +168,10 @@ public async Task<GetTransportSettingsResponse> GetTransportSettingsAsync()
 
             return new GetTransportSettingsResponseFactory().CreateFor(ActionXNamespace, xml);
         }
+
+        public async Task SetPlayModeAsync()
```
Comment (ByteDev) [inline src/ByteDev.Sonos.Upnp/Services/AvTransportService.cs]: @ifcodesdotnet Hi, what's the point of this method? It isn't called and doesn't supply a PlayMode type like the existing overload above. Was accidental commit? :-) Thanks

Comment (ifcodesdotnet) [reply src/ByteDev.Sonos.Upnp/Services/AvTransportService.cs]: I've committed a change to remove the unnecessary method! Sorry !

**Comment by ifcodesdotnet** [comment]
I've removed the SetPlayModeAsync method that isn't called, please review the change and let me know if you find anymore issues. Thank you!

**Comment by ByteDev** [comment]
This change is now in v1.2.0 on nuget.

[https://www.nuget.org/packages/ByteDev.Sonos/1.2.0](https://www.nuget.org/packages/ByteDev.Sonos/1.2.0)

Thanks

**Pull request merged**